src/components/avatar/avatar.module.scss

```diff
@@ -0,0 +1,23 @@
+:root {
+    --avatarSize: 22px;
+}
+
+.root {
+    width: var(--avatarSize);
+    height: var(--avatarSize);
+    overflow: hidden;
+
+    display: flex;
+    align-items: center;
+    border-radius: var(--avatarSize);
+}
```

Comment:
Didn't test, but something like this should work:

```
.root {
    width: 22px;
    height: 22px;
    color: var(--primary5);
}

.root :is(img, svg) {
   display: block;
   width: 100%;
   height: 100%;
}

.root img {
    object-fit: cover;
    border-radius: 50%;
}
```

Reply:
applied, but I used `object-fit: fill;` to better handle non-circle avatars, like codux logo